🐛 BUG: tokens for reading uptime checks throw a scary error when adding new checks

### What version of `onlineornot` are you using?

latest

### What operating system are you using?

Linux

### Describe the Bug

```
rozenmd@thinkpad-pro:~$ onlineornot checks create test https://google.com
✘ [ERROR] ParseError: A request to the OnlineOrNot API (/checks/) failed.

      at throwFetchError
  (/home/rozenmd/.nvm/versions/node/v16.19.0/lib/node_modules/onlineornot/onlineornot-dist/cli.js:25406:17)
      at fetchResult
  (/home/rozenmd/.nvm/versions/node/v16.19.0/lib/node_modules/onlineornot/onlineornot-dist/cli.js:25377:5)
      at processTicksAndRejections (node:internal/process/task_queues:96:5)
      at async Object.handler
  (/home/rozenmd/.nvm/versions/node/v16.19.0/lib/node_modules/onlineornot/onlineornot-dist/cli.js:25452:14)
  {
    text: 'A request to the OnlineOrNot API (/checks/) failed.',
    notes: [ { text: 'Unauthorized [code: 10003]' } ],
    location: undefined,
    kind: 'error',
    code: 10003
  }
```

--- a/.changeset/smart-files-work.md
+++ b/.changeset/smart-files-work.md
@@ -1,0 +1,5 @@
+---
+"onlineornot": patch
+---
+
+fix: improve error when token isn't authorized to create checks

--- a/packages/onlineornot/src/checks/create.ts
+++ b/packages/onlineornot/src/checks/create.ts
@@ -47,6 +47,11 @@ export async function handler(
 			return logger.error(
 				"You have reached the maximum number of checks for your account. Please upgrade to a paid plan to add more checks."
 			);
+		} else if (errorWithCode.code === 10003) {
+			//unauthorized
+			return logger.error(
+				"Your API token isn't allowed to create checks.\nPlease check your token with `onlineornot whoami` and try again."
+			);
 		} else if (errorWithCode.code === 10000) {
 			//validation error, need to check notes
 			return logger.error(

